mizar/store/operator_store.py

```diff
@@ -128,6 +133,14 @@ def delete_ep(self, name):
         l = len(self.eps_net_store[ep.net])
         if l == 0:
             del self.eps_net_store[ep.net]
+
+        if ep.k8s_pod_name in self.eps_pod_store.keys():
+            if name not in self.eps_pod_store[ep.k8s_pod_name]:
+                return ep
+            self.eps_pod_store[ep.k8s_pod_name].pop(name)
+            l = len(self.eps_pod_store[ep.k8s_pod_name])
```

Comment:
Didn't you delete the "k8s_pod_name" field from the endpoint in the latest commit?

Reply:
Yes, thanks for catching this.